test/karpenter.test.ts

```diff
@@ -73,6 +73,30 @@ describe('Unit tests for Karpenter addon', () => {
         }).toThrow("Consolidation and ttlSecondsAfterEmpty must be mutually exclusive.");
     });
 
+    test("Support Windows2019 as amiFamily", () => {
```

Comment:
please make sure github warnings are addressed here, I suppose running `make lint` should be enough to get the errors.